src/main/java/org/openpnp/vision/pipeline/stages/ImageCapture.java

```diff
@@ -1,76 +1,77 @@
-package org.openpnp.vision.pipeline.stages;
-

```

Comment:
Hi Tony

Do you know why it does this? Marking everything as changed? Some newline issue?

I suspect, this will create artificial conflicts with #1106 etc. that are in testing. Could you please resolve this? Thanks!

_Mark

Reply:
It appears the current file in the repository has CRLF line endings but my changed file has LF line endings.   I believe LF is the correct line ending.  Looks like the previous change to the file introduced the wrong line endings.

Tony